code/game/objects/items/implants/implant_bb.dm

```diff
@@ -0,0 +1,65 @@
+/obj/item/implant/bloodbrother
+	name = "communication implant"
+	desc = "Use this to communicate with your fellow blood brother(s)."
+	icon = 'icons/obj/radio.dmi'
+	icon_state = "headset"
+	item_color = "r"
+	var/list/linked_implants // All other implants that this communicates to
+
+/obj/item/implant/bloodbrother/Initialize()
+	. = ..()
+	linked_implants = list()
+
+/obj/item/implant/bloodbrother/activate()
+	. = ..()
+	if(linked_implants.len)
+		var/input = stripped_input(imp_in, "Enter a message to communicate to your blood brother(s).", "Radio Implant", "")
+		if(!input)
+			return
+
+		var/preliminary_message = "<span class='bold'>[input]</span>" //apply basic color/bolding
+		var/my_message = "<font color=\"#ff0000\"><b><i>[imp_in]:</i></b></font> [preliminary_message]" //add source, color source with syndie color
+		var/ghost_message = "<font color=\"#ff0000\"><b><i>[imp_in] -> Blood Brothers:</i></b></font> [preliminary_message]"
+
+		to_chat(imp_in, my_message) // Sends message to the user
+		for(var/obj/item/implant/bloodbrother/i in linked_implants) // Sends message to all linked implnats
+			var/M = i.imp_in
+			to_chat(M, my_message)
+		for(var/M in GLOB.dead_mob_list) // Sends message to ghosts
+			var/link = FOLLOW_LINK(M, imp_in)
+			to_chat(M, "[link] [ghost_message]")
+
+		imp_in.log_talk(input, LOG_SAY, tag="Blood Brother Implant")
+	else
+		to_chat(imp_in, "<span class='bold'>There are no linked implants!</span>")
+
+/obj/item/implant/bloodbrother/Destroy()
+	. = ..()
+	for(var/obj/item/implant/bloodbrother/i in linked_implants) // Removes this implant from the list of implants
+		i.linked_implants -= src
+
+/obj/item/implant/bloodbrother/proc/link_implant(var/obj/item/implant/bloodbrother/BB)
+	if(BB)
+		if(BB == src) // Don't want to put this implant into itself
+			return
+		if(locate(BB) in linked_implants || locate(src) in BB.linked_implants) // Don't want duplicates in the list
+			return
+		linked_implants += BB
+		BB.linked_implants += src
```

Comment:
```suggestion
		linked_implants |= BB
		BB.linked_implants |= src
```
Can't you compress it down to this?

Reply:
yes you can, I didn't even know about this operator. thanks